index.html

```diff
@@ -82,6 +82,7 @@
 // production code goes here</textarea>
     <p id="status"></p>
     <script type="text/javascript">
+(function iife() {
```

Comment:
Sure! Figured it might be intentional, which is why this is split in two commits. 👃 👈